src/codegen/tests/parameters_tests.rs

```diff
@@ -495,59 +442,149 @@ fn program_empty_inout_assignment() {
 }
 
 #[test]
-fn program_missing_input_assignment() {
+fn program_accepts_empty_statement_as_input_param() {
     // GIVEN
     let result = codegen(
         "
 		PROGRAM prog
 		VAR_INPUT
-			input1 : DINT;
+			in1: DINT;
+			in2: DINT;
 		END_VAR
+		END_PROGRAM
+
+		PROGRAM main
+			prog(in1 := 1, in2 := );
+		END_PROGRAM
+		",
+        // TODO: in assignments for stateless POUs (x := ), see how omitting the parameter is handled and copy
```

Comment:
What's the status for this TODO?

Reply:
That was a comment to myself on how I might approach this fix and just forgot to delete it.